CHANGES/5627.removal

```diff
@@ -0,0 +1 @@
+Remove plugin managed repos
```

Comment:
@fabricio-aguiar i think we should add .feature where we'd capture the new field '`sub_repo`